app/pulp/app/viewsets/repository.py

```diff
@@ -72,6 +72,15 @@ class ImporterViewSet(NamedModelViewSet):
     serializer_class = ImporterSerializer
     endpoint_name = 'importers'
 
+    def destroy(self, request, name):
+        imp = self.get_object()
+        async_result = importer.delete.apply_async_with_reservation(
+            tags.RESOURCE_REPOSITORY_IMPORTER_TYPE, imp.name,
+            kwargs={'repo_name': imp.repository,
+                    'importer_name': imp.name}
```

Comment:
Do we want to add tags here?  At least they were [specified in pulp 2](https://github.com/pulp/pulp/blob/3.0-dev/server/pulp/server/controllers/importer.py#L184) .